modules/react/table/stories/stories_visualTesting.tsx

```diff
@@ -74,34 +75,58 @@ const FixedColumn = () => (
         <Table.Row>
           <Table.Header
             position="sticky"
-            left="0"
-            backgroundColor="soap100"
-            borderRight={`1px solid ${colors.soap400}`}
+            left={system.space.zero}
+            backgroundColor={system.color.bg.alt.softer}
+            borderRight={`1px solid ${cssVar(system.color.border.divider)}`}
```

Comment:
I know it's just visual tests, but would it be better if we use our styling functions for this instead of style props? I feel it helps with the future migration

Reply:
Wouldn't hurt to make the update!